internal/functions/Get-DecryptedObject.ps1

```diff
@@ -22,7 +22,7 @@ function Get-DecryptedObject {
     # Query Service Master Key from the database - remove padding from the key
     # key_id 102 eq service master key, thumbprint 3 means encrypted with machinekey
     Write-Message -Level Verbose -Message "Querying service master key"
-    $sql = "SELECT substring(crypt_property,9,len(crypt_property)-8) as smk FROM sys.key_encryptions WHERE key_id=102 and (thumbprint=0x03 or thumbprint=0x0300000001)"
+    $sql = "SELECT substring(crypt_property,9,datalength(crypt_property)-8) as smk FROM sys.key_encryptions WHERE key_id=102 and (thumbprint=0x03 or thumbprint=0x0300000001)"
```

Comment:
What purpose does this change serve?

Reply:
LEN returns the number of characters in the string, DATALENGTH returns the number of bytes.

Reply:
i just see it here and did a test on my env and it works:
https://github.com/bugal/Powershell-Modules/commit/606153b890f831e94890ef85793451b28ead0433

there are other fix to workground for SQL cluster too ,you may need this https://github.com/NetSPI/Powershell-Modules/pull/9